src/components/ModalComponent.tsx

```diff
@@ -24,12 +25,16 @@ interface ModalComponentProps {
 
 @autobind
 export class ModalComponent extends React.Component<ModalComponentProps, any> {
+
+    private modalBodyRef: any
+
     constructor(props: any) {
         super(props)
         this.state = {
             item: this.props.item || {},
             deleteModal: false
         }
+        this.modalBodyRef = React.createRef()
```

Comment:
we are overriding this, what will this call do?

Reply:
It helps us to store, the html node of that modal. So that we can add native html features

Reply:
Updated, without React.createRef